DataTask/datapush/uploadJsonToCos/app.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright 2022 IBM Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import uuid
+from flask import Flask, Blueprint, render_template, request, escape, jsonify
+from flask_swagger import swagger
+from swagger_ui_bundle import swagger_ui_path
+from flask_restx import Api, Resource
+from werkzeug.datastructures import FileStorage
+from werkzeug.middleware.proxy_fix import ProxyFix
+from werkzeug.utils import secure_filename
+import pandas as pd
+import json
+import constants
+import requests
+import ibm_boto3
+from ibm_botocore.client import Config, ClientError
+import hashlib
+import glob
+
+
+app = Flask(__name__)
+api = Api(app)
+
+upload_parser = api.parser()
+upload_parser.add_argument('file', type=FileStorage, location='files',required=True)
+upload_parser.add_argument('data_repository_configuration_ID', type=str,required=True)
+
+path = os.getcwd()
+# file Upload
+UPLOAD_FOLDER = os.path.join(path, 'pushed_datasets')
+
+if not os.path.isdir(UPLOAD_FOLDER):
+    os.mkdir(UPLOAD_FOLDER)
+
+app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+ALLOWED_EXTENSIONS = {'csv', 'json'}
+
+def allowed_file(filename):
+    return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+# Clean up function to remove files created under the pushed_datasets folder
+def delete_if_exists(url):
+    os.remove(url)  
+
+@api.route('/datapush')
+@api.expect(upload_parser)
+class CollectData(Resource):
```

Comment:
In practice, app.py should not have long class definitions. Having a class in another file and creating an instance of the class here is recommended. This is just a recommendation, the PR can be merged as it is now.

Reply:
I'll work on it!